README.md

```diff
@@ -83,7 +83,8 @@ Please see [MinikubeDemo.md](./examples/MinikubeDemo.md) for more details.
 ## Web UI
 
 Katib provides a Web UI.
-You can visualize general trend of Hyper parameter space and each training history.
+You can visualize general trend of Hyper parameter space and each training history. The following example UI
+is generated by [random-example](https://github.com/kubeflow/katib/blob/master/examples/random-example.yaml).
```

Comment:
Thanks! But the example UI is not generated by the example (studyjob name and requestcount don't match at least).

Reply:
@hougangliu Updated a bit to tell end user can use the example to generate a similar UI. ;-)